.pre-commit-config.yaml

```diff
@@ -190,17 +190,17 @@ repos:
         # is not available via setup.py and only (unofficially)
         # available in docker.
         entry: -e SEMGREP_LOG_FILE=/tmp/out.log -e SEMGREP_VERSION_CACHE_PATH=/tmp/cache returntocorp/semgrep:develop semgrep
-        exclude: "^semgrep-core/src/pfff"
-        #note that the --exclude below are useless because pre-commit pass
-        # a list of files explicitely which prevents --exclude to work
-        # hence the separate exclude: above.
+
+        # Both the .semgrepignore file and the --exclude option
```

Comment:
Maybe add a TODO that says that once we redesign file targeting in (o)semgrep
we should not need that anymore.
In the mean time we have to use the pre-commit exclude mechanism because we can't rely
on the semgrep ignore mechanism (bypassed by passing files explicitely by pre-commit).